resources/views/auth/register.blade.php

```diff
@@ -25,13 +25,15 @@
                                 @include("auth.register.contact")
                             </div>
                         </div>
-
-						<div class="card">
-							<div class="card-header">@lang('info.information_of_studies')</div>
-							<div class="card-body">
-                                @include("auth.register.information_of_studies")
+                        
+                        @if(!($isTenant ?? false))
+                            <div class="card">
+                                <div class="card-header">@lang('info.information_of_studies')</div>
+                                <div class="card-body">
+                                    @include("auth.register.information_of_studies")
+                                </div>
                             </div>
-                        </div>
```

Comment:
This div here should not be removed.

Reply:
I don't see why. Didn't just the github formatted it strangely here?

Reply:
You are right, my bad.